core/src/main/scala/filodb.core/memstore/TimeSeriesPartition.scala

```diff
@@ -322,7 +322,7 @@ extends ChunkMap(memFactory, initMapSize) with ReadablePartition {
 
   final def earliestTime: Long = {
     if (numChunks == 0) {
-      Long.MinValue
+      Long.MaxValue
```

Comment:
Given that you simply flipped this from min to max confuses me. Can you add a doc regarding the method contract?

Reply:
Added doc.
We use earliestTime method in new contexts now which need an alternate default value if chunks are not present.
Hence I had to change the contract.